controllers/dspipeline_params.go

```diff
@@ -235,7 +239,15 @@ func (p *DSPAParams) SetupObjectParams(ctx context.Context, dsp *dspa.DataScienc
 		p.ObjectStorageConnection.Bucket = dsp.Spec.ObjectStorage.ExternalStorage.Bucket
 		p.ObjectStorageConnection.Host = dsp.Spec.ObjectStorage.ExternalStorage.Host
 		p.ObjectStorageConnection.Scheme = dsp.Spec.ObjectStorage.ExternalStorage.Scheme
-		p.ObjectStorageConnection.Secure = dsp.Spec.ObjectStorage.ExternalStorage.Secure
+
+		if dsp.Spec.ObjectStorage.ExternalStorage.Secure == nil {
+			if p.ObjectStorageConnection.Scheme == "https" {
+				p.ObjectStorageConnection.Secure = BoolPointer(true)
+			} else {
+				p.ObjectStorageConnection.Secure = BoolPointer(false)
+			}
+		}
```

Comment:
This won't account for when `secure` _is set_, in which case it will be nil and thus crash the server, can you update it to the following: 

```go
if dsp.Spec.ObjectStorage.ExternalStorage.Secure == nil {
	if p.ObjectStorageConnection.Scheme == "https" {
		p.ObjectStorageConnection.Secure = BoolPointer(true)
	} else {
		p.ObjectStorageConnection.Secure = BoolPointer(false)
	}
} else {
	p.ObjectStorageConnection.Secure = dsp.Spec.ObjectStorage.ExternalStorage.Secure
}
```

Reply:
That was a big miss....re added that else in the PR.